stream_alert/apps/_apps/README.rst

```diff
@@ -96,3 +96,66 @@ back on local system
 
   # scp to local host's current directory
   $ scp -i /path/to/<private-key>.pem ec2-user@public.dns.address:~/pip_temp/pip.zip .
+
+Build dependencies on an virtual machine locally (python3.7)
```

Comment:
super long header.. maybe:

`Building Dependencies Using Vagrant`

Reply:
I like long long header 🤣

Reply:
pls no. it looks bad when rendered in docs